treeview/js/main.js

```diff
@@ -5,6 +5,8 @@
         // console.log($);
         // console.log(ERMrest);
         // console.log(Q);
+        var urlParams=[];
+        var urlParamNames=[];
```

Comment:
please don't redefine these properties here. Remove the definition and uncomment the ones below.

Reply:
Made required changes